File: db/password-utils.js
import crypto from 'crypto';

// Hàm mã hóa mật khẩu - Lưu trực tiếp không hash
export function hashPassword(password) {
  return password; // Trả về mật khẩu gốc
}

// Hàm kiểm tra mật khẩu - So sánh trực tiếp
export function verifyPassword(password, storedPassword) {
  try {
    return password === storedPassword; // So sánh trực tiếp
  } catch (error) {
    console.error('Lỗi khi kiểm tra mật khẩu:', error);
    return false;
  }
}

// Hàm tạo hash với salt cố định (để test) - Không cần thiết nữa
export function hashPasswordWithSalt(password, salt) {
  return password; // Trả về mật khẩu gốc
} 

File: test_system/password_utils.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

def hash_password(password):
    """
    Lưu mật khẩu trực tiếp không hash
    """
    return password

def verify_password(password, stored_password):
    """
    Kiểm tra mật khẩu - So sánh trực tiếp
    """
    try:
        return password == stored_password
    except Exception as e:
        print(f"Lỗi khi kiểm tra mật khẩu: {e}")
        return False

def hash_password_with_salt(password, salt):
    """
    Lưu mật khẩu trực tiếp không hash (không cần thiết nữa)
    """
    return password 